virtual_ecosystem/data_variables.toml

```diff
@@ -1,31 +1,23 @@
+[[variable]]
+name = "air_heat_conductivity"
+description = "Air heat conductivity between layers"
+unit = "mol m-2 s-1"
+variable_type = "DataArray(float)"
+axis = [space]
+
 [[variable]]
 name = "air_temperature"
 description = "Air temperature profile"
 unit = "C"
-variable_type = ""
-axis = []
+variable_type = "DataArray(float)"
```

Comment:
I'm not sure if we want to indicate that this is a `DataArray` of floats or just indicate `floats`, taking for granted that they are all going to be `DataArray`s. Totally up to you and the use case you have in mind.

Reply:
Could you please remind me what exactly this `variable_type` is used for or validated against? If all entries are DataArrays, then I can easily change in to floats. If we have some very different types, like the functional groups or so, I would prefer keeping a more complete description. Thoughts @davidorme @jacobcook1995 ?

Reply:
I think just `float`. It's more compact and it is also easier to use the value directly in checking, if we wanted to use the variables system to type check inputs.

I don't think we'd be loading functional groups by this method - if we do implement a `read_csv` for dataframes, then the fields should be loaded into `Data` as individual `DataArrays`. Anything dict/toml like would come in by a bespoke route or config.